### DISCUSSION TITLE
Trying to draw Rect

### DISCUSSION THREAD

**Discussion by QM4RS** [vfsfitvnm/frida-il2cpp-bridge#397] [Q&A]
HI. I'm trying to make a menu with Frida-il2cpp-bridge.

currently, I am trying to do this:
`const IMGUIModule = Il2Cpp.domain.assembly("UnityEngine.IMGUIModule").image`
`const CoreModule = Il2Cpp.domain.assembly("UnityEngine.CoreModule").image`
`const GUIStyle = IMGUIModule.class("UnityEngine.GUI")`
`const GUIContent = IMGUIModule.class("UnityEngine.GUIContent")`
`const Rect = CoreModule.class("UnityEngine.Rect")`
    
`const Temp = GUIContent.method<Il2Cpp.Object>("Temp").invoke(Il2Cpp.string("TEST"))`
`const my_rect = Rect.method<Il2Cpp.Object>("MinMaxRect").invoke(20,20,20,20)`
`Il2Cpp.gc.choose(GUIStyle)[0].method("Draw").invoke(my_rect,Temp,1234,false,false,false)`

Given Error:
access violation accessing 0xd6c

**Comment by VAROME** [comment]
you can just fix one of these and work with it 
https://[github.com/commonuserlol/frida-java-menu](https://github.com/commonuserlol/frida-java-menu)
https://github.com/maarsalien/frida-android-mod-menu
